README.md

```diff
@@ -1,8 +1,48 @@
-# backend-main
-1. starting project: 
-    django-admin startproject <name_project>
-2. starting application
-    python manage.py startapp <name_application>
-3. registry application at settings.py on folder project main with variable "INSTALLED_APPS"
-4. install all of dependencies with:
-    pip install -r requirement.txt
\ No newline at end of file
+# Backend Main
+This is an API that contains the main functions. This will be forked into our main backend project.
+
+## Repository Structure
+Here is the overview of the repository's structure.
+```
+.
+├── _main
+│   ├── __init__.py
+│   ├── __pycache__
+│   ├── asgi.py
+│   ├── settings.py
+│   ├── urls.py
+│   └── wsgi.py
+├── .gitignore
+├── LICENSE
+├── manage.py
+├── README.md
+├── requirement.txt
+└── <your-apps-directory>
+```
+
+## Getting Started
+### 1. Create Virtual Environment
+Let's start this learn by creating your virtual environment (venv). Please refer to this link to see how to make it: https://gist.github.com/ryumada/c22133988fd1c22a66e4ed1b23eca233
+
+### 2. Install The Packages in `requirement.txt`
+```bash
+pip install -r requirement.txt
+```
+
+### 3. Start The Development by Creating a New App
+```bash
+django-admin startapp app-name
+```
+
+### 4. Register the Packages and the Applications
```

Comment:
This line, I add the steps to place the installed package to be used in the app.